iot-hub/Samples/device/ConventionBasedOperations/Thermostat/ThermostatSample.cs

```diff
@@ -83,12 +86,18 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
             }
         }
 
-        private async Task GetWritablePropertiesAndHandleChangesAsync()
+        private async Task GetWritablePropertiesAndHandleChangesAsync(CancellationToken cancellationToken)

```

Comment:
have you considered adding the property name as a parameter? it will be easier to reuse this method if there is more than one property to initialize

Reply:
`GetWritablePropertiesAndHandleChangesAsync` is called only on `ConnectionStatus.Connected`, so it is more reasonable to set "propertyName"/ "componentName" as parameter for the method `UpdateForEmptyWritableProperty` which is implemented in TemperatureController sample. I can do the same for Thermostat though.